### PR TITLE
Less noisy auth in release action

### PR DESCRIPTION
**Pull request by mallport**
Inspired by [data-collector-api workflow](https://github.com/statisticsnorway/data-collector-api/blob/master/.github/workflows/mvn-release.yml)

This change is [Reviewable](https://reviewable.io/reviews/statisticsnorway/maskinporten-guardian/70)

### CODE REVIEW

File: .github/workflows/mvn-release.yml [hunk 1]
```diff
@@ -50,25 +50,21 @@ jobs:
 
       - name: Configure Git user
         run: |
-          git config user.email "dapla-bot[bot]"
-          git config user.name "143391972+dapla-bot[bot]@users.noreply.github.com"
+          git config user.email "143391972+dapla-bot[bot]@users.noreply.github.com"
```
Comment (kschulst) [inline .github/workflows/mvn-release.yml:53]: We might want to use an env variable for the name and email of the Bot.

**Review by skykanin — APPROVED** [review]

**Review by kschulst — APPROVED** [review]
See suggestion about using env variables for the dapla bot email/name

**Comment by mallport** [comment]
Deprecated by https://github.com/statisticsnorway/maskinporten-guardian/pull/71

**Comment by mallport** [comment]
Deprecated by https://github.com/statisticsnorway/maskinporten-guardian/pull/71

**Pull request closed without merging**